shared/dev/dumb-sheet.render.native.js

```diff
@@ -36,8 +37,36 @@ class Render extends Component<void, Props, any> {
     }, 0)
   }
 
+  _increment () {
+    if (this.props.autoIncrement && this.state.testIndex !== -1) {
+      const {componentsOnly} = this._getComponents()
+      if (this.state.testIndex >= componentsOnly.length) {
+        this.setState({testIndex: -1})
+      } else {
+        this.setState({testIndex: this.state.testIndex + 1})
+      }
+    }
+  }
+
+  componentDidMount () {
+    if (this.props.autoIncrement) {
+      this._increment()
+    }
+  }
+
+  componentDidUpdate () {
+    if (this.props.autoIncrement) {
```

Comment:
:sheep: Don't want to block this from moving forward, but I still think this is too much behavior in the render component. Ideally the index of the sheet displayed should be controlled by the container via props.